api/envoy/config/route/v3/route_components.proto

```diff
@@ -453,6 +453,20 @@ message WeightedCluster {
   }
 }
 
+// Configuration for a cluster specifier plugin.
+message ClusterSpecifierPlugin {
```

Comment:
I think moving this message from one file to another is technically a breaking API change, because it affects the generated code and may break builds.

@dfawley @ejona86 Can grpc-go and grpc-java tolerate this change, or is it going to cause breakage?

Reply:
This moving will not change this full name of this type. And it will not introduce additional deps, so it should be safe.

Reply:
Go is fine with this change, since it stays within the same generated code package.  I'm not sure about Java.

Reply:
Java should also be ok. `route.proto` depends on `route_component.proto`. So it should be safe to move a message from `route.proto` to `route_component.proto`. 🤔